services/common/src/interfaces/common/match.interface.ts

```diff
@@ -0,0 +1,6 @@
+export interface IMatch {
```

Comment:
I would say this is actually not necessary. In MineSummary, for example, you can replace
`const { id } = this.props.match.params`
with
`const { id } = useParams();`
and just get rid of the prop

Reply:
Yah looking at this again the IMatch really isn't necessary. It has been removed now and useParams() is used within MineSummary instead.

Reply:
Nice!
I see the tests are giving you trouble- we've got a few examples of mocking this in MineIncident.spec, noticeOfDeparture.spec, and ProjectSummaryPage.spec

Reply:
Thanks for mentioning the examples, that helped a lot